src/server/api/lib/twilio.js

```diff
@@ -159,14 +163,14 @@ async function sendMessage(message) {
 async function handleDeliveryReport(report) {
   const messageSid = report.MessageSid
   if (messageSid) {
+    await Log.save({ message_sid: report.MessageSid, body: report })
     const messageStatus = report.MessageStatus
     const message = await r.table('message')
       .getAll(messageSid, { index: 'service_id' })
       .limit(1)(0)
       .default(null)
-
     if (message) {
-      message.service_response += JSON.stringify(report)
```

Comment:
this may save storage space for all the tech info, but worth it?  or is this saved above somehow?  My vote would be to keep `message.service_response`

Reply:
the concat of json blocks makes it not possible to programatically report on latency in system
we've move these responses to the log table